src/lib/Form/Factory/FormFactory.php

```diff
@@ -1330,4 +1332,26 @@ public function trashContainerLocation(
 
         return $this->formFactory->createNamed($name, LocationTrashContainerType::class, $data);
     }
+
+    /**
+     * @param MainTranslationUpdateData|null $data
+     * @param string|null $name
+     *
+     * @return FormInterface
+     *
+     * @throws InvalidOptionsException
+     */
+    public function updateMainTranslation(
```

Comment:
It has been decided recently that we are no longer adding new methods to the `FormFactory` and we will use Symfony's FormFactory for this.